tests/OutputTest/Dataset.php

```diff
@@ -0,0 +1,94 @@
+<?php
+namespace Dompdf\Tests\OutputTest;
+
+use Dompdf\Options;
+use Dompdf\Dompdf;
+use SplFileInfo;
+
+class Dataset
+{
+    public const DATASET_DIRECTORY = __DIR__;
+
+    /**
+     * @var SplFileInfo
+     */
+    protected $file;
+
+    public function __construct(SplFileInfo $file)
```

Comment:
I think it's good to put a comment here explaining that `$file` is the source HTML file.